build.test.ts

```diff
@@ -1,19 +1,47 @@
-import { loadTemplate, loadData, Template, Data, renderTemplates } from "./build";
-import fs from "fs-extra"
+import {
+	loadTemplate,
+	loadData,
+	loadTemplatePartials,
+	Template,
+	Data,
+	renderTemplates,
+} from './build'
+import fs from 'fs-extra'
+import path from 'path'
 
-let templates: Template[];
-let data: Data;
+jest.setTimeout(120000)
+
+let templates: Template[]
+let data: Data
+let partials: Data
 describe('build', () => {
 	test('should load template', () => {
-		templates = loadTemplate();
-		expect(templates.length).toBeGreaterThan(0);
-	});
+		templates = loadTemplate()
+		expect(templates.length).toBeGreaterThan(0)
+	})
 	test('should load data', async () => {
-		data = await loadData();
-		expect(data).toBeTruthy();
-	});
+		data = await loadData()
+		expect(data).toBeTruthy()
+	})
+	test('should load partials', async () => {
+		partials = loadTemplatePartials()
+		expect(partials).toBeTruthy()
+	})
 	test('should gernerate html', () => {
-		renderTemplates(templates, data);
-		expect(fs.readFileSync('./site/index.html', 'utf8')).toBeTruthy();
-	});
-});
\ No newline at end of file
+		renderTemplates(
+			[
+				{
+					path: path.resolve('./pages'),
+					name: 'index',
+					content: 'hello, world!',
```

Comment:
If I attempt to serve `./site/` after running the test suite, I get a page that says `hello, world!`. It might make sense to render this into a test/temporary directory instead.

Reply:
I‘ve changed the output path for tests, it won't override the original output files anymore.

Reply:
Thank you! I'm testing it again!

Reply:
It works!